src/sphinxutils.cpp

```diff
@@ -802,18 +802,19 @@ static KeyDesc_t g_dKeysIndex[] =
 /// allowed keys for indexer section
 static KeyDesc_t g_dKeysIndexer[] =
 {
-	{ "mem_limit",				0, NULL },
```

Comment:
We try to distinguish actual changes from syntax changes. Can you please revert this # of tabs change? If you feel it's important and if it complies with our [coding standards](https://github.com/manticoresoftware/manticoresearch/blob/master/doc/internals-coding-standard.txt) please commit it separately.